packages/components/addon/components/hds/tooltip/index.hbs

```diff
@@ -0,0 +1,24 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Interactive
+  class="hds-tooltip"
+  {{hds-tooltip this.text options=this.options}}
+  @current-when={{@current-when}}
+  @models={{hds-link-to-models @model @models}}
+  @query={{hds-link-to-query @query}}
+  @replace={{@replace}}
+  @route={{@route}}
+  @isRouteExternal={{@isRouteExternal}}
+  @href={{@href}}
+  @isHrefExternal={{@isHrefExternal}}
+  ...attributes
+>
+  {{#if @icon}}
+    <FlightIcon class="hds-tooltip__icon" @name={{@icon}} />
+  {{else}}
+    {{~yield~}}
+  {{/if}}
+</Hds::Interactive>
```

Comment:
I'm experimenting with an API as explained here: https://docs.google.com/document/d/1VurJJ7itbO0NtAKzAN4sxmQDLgx7yzGf8-TfFmBptNc/edit?disco=AAAAs--9950

Reply:
After discussion with @didoo I'm going to go back to my original simpler implementation using just a plain button vs. `Hds::Interactive` and also remove the `@icon` arg and instead just use `yield`.

Reply:
The rationale was mainly to simplify the component. And since we are providing both the component and the modifier (helper), other use cases can be handled by the modifier.